### PR TITLE
Add node list json

### PR DESCRIPTION
**Pull request by Kay-Zee**
## Description

Adds the node list json used to end Epochs

______

For contributor use:

- [x] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

**Review by laynelafrance — COMMENTED** [review]
@Kay-Zee thoughts on me updating this into table format to add onto this? https://github.com/onflow/flow/blob/master/nodeoperators/NodeOperatorList.md

File: nodeoperators/nodelist.json [hunk 1]
```diff
@@ -0,0 +1,2826 @@
+[
+    {
+        "Role": "verification",
+        "Address": "verification-009.mainnet4.nodes.onflow.org:3569",
```
Comment (vishalchangrani) [inline nodeoperators/nodelist.json:4]: shouldn't these be `mainnet5` nodes

**Comment by Kay-Zee** [comment]
@laynelafrance what do you mean by update it into table format? Theres currently the table, and then there is this JSON file that's used during end epoch. You want to combine them?

File: nodeoperators/nodelist.json (hunk 1 above)
Comment (Kay-Zee) [reply nodeoperators/nodelist.json:4]: they're stuck as mainnet4 in the smart contract, but i guess i can update them here

**Comment by laynelafrance** [comment]
Nvm @Kay-Zee - Vishal explained - all good - approving

**Review by laynelafrance — APPROVED** [review]

**Pull request merged**